tools/zbkey

```diff
@@ -97,16 +107,16 @@ arg_delay = 4
 # Parameters to build association packet
 arg_IEE64addr = None    #May be specified on command line
 arg_panID = None        #May be specified on command line
-ASSOC_SEQNUM = '\x01'
+ASSOC_SEQNUM = '\x70'
```

Comment:
What is the logic for these changes? Did you experimentally see more success with higher sequence numbers or was this specific to the target you were working on?